paasta_tools/kubernetes_tools.py

```diff
@@ -724,6 +724,19 @@ def get_aws_ebs_volume_name(self, aws_ebs_volume: AwsEbsVolume) -> str:
     def read_only_mode(self, d: VolumeWithMode) -> bool:
         return d.get("mode", "RO") == "RO"
 
+    def get_readiness_check_script(
+        self, system_paasta_config: SystemPaastaConfig
+    ) -> str:
+        """Script to check if a service is up in smartstack / envoy"""
+        enable_envoy_check = self.get_enable_envoy_readiness_check(system_paasta_config)
+        enable_nerve_check = self.get_enable_nerve_readiness_check(system_paasta_config)
+        if enable_nerve_check and enable_envoy_check:
+            return "/check_proxy_up.sh --enable-smartstack --enable-envoy"
```

Comment:
The check scripts themselves could be configured in the conf (`system_paasta_config.get_nerve_readiness_check_script()`). Are we ok with hardcoding them here instead?

At least we are building the sidecar in the same repo so they shouldn't drift apart given testing.

Reply:
np, I'll add these scripts as params on system_paasta_config.